docs/source/en/package_reference/environment_variables.md

```diff
@@ -74,6 +74,14 @@ small files will be duplicated to ease user experience while bigger files are sy
 
 For more details, see the [download guide](../guides/download#download-files-to-local-folder).
 
+### HF_HUB_ETAG_TIMEOUT
+
+Integer value to define etag timeout when fetching ETag on the system level.
+
+### HF_HUB_DOWNLOAD_TIMEOUT
+
+Integer value to define get request timeout when fetching a file on the system level.
```

Comment:
```suggestion
Integer value to define the number of seconds to wait for server response when downloading a file. If the request times out, a TimeoutError is raised. Setting a higher value is beneficial on machine with a slow connection. A smaller value makes the process fail quicker in case of complete network outage. Default to 10s.
```